misk-zookeeper/src/main/kotlin/misk/zookeeper/CuratorFrameworkProvider.kt

```diff
@@ -47,6 +68,24 @@ internal class CuratorFrameworkProvider @Inject internal constructor(
           }
           ZooKeeper(connectString, sessionTimeout, watcher, canBeReadOnly, clientConfig)
         }
+        .aclProvider(object : ACLProvider {
+          override fun getDefaultAcl(): List<ACL> {
+            // Default ACL allows clients CRUD operations on znodes this service has created.
+            // Service identification comes from the client cert
+            return Collections.singletonList(ACL(DEFAULT_PERMS, AUTH_IDS))
+          }
+
+          override fun getAclForPath(path: String?): List<ACL> {
+            // Shared directories need to be created in such a way that anyone can create them if
+            // they're missing and read or write to them, but anyone should not be able to delete
```

Comment:
lol, all good! An individual service shouldn't be able to delete a shared directory like `services` -- because that would delete all the service-specific subdirectories.